src/test/java/edu/ucsb/courses/documents/statistics/SingleCourseSearchTests.java

```diff
@@ -0,0 +1,70 @@
+package edu.ucsb.courses.documents.statistics;
+
+import java.util.List;
+import net.codebox.javabeantester.JavaBeanTester;
+import org.aspectj.apache.bcel.classfile.Module;
+import org.junit.jupiter.api.BeforeEach;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+public class SingleCourseSearchTests {
+
+    private static Logger logger = LoggerFactory.getLogger(QuarterDeptTests.class);
+    private SingleCourseSearch test1, test2;//, scs;
```

Comment:
The variable `scs` is used in subsequent tests. Should this be uncommented? If this is uncommented, line 22 can also be uncommented, furthermore we can then delete lines 38 and 44.

Reply:
Fixed the bug, and resolved the variable issue.